app/javascript/orangelight/availability.es6

```diff
@@ -189,12 +203,20 @@ export default class AvailabilityUpdater {
     })();
   }
 
-  // This method is used to set the availability info to Undetermined when
-  // the call to the Availability endpoint fails.
-  update_single_undetermined() {
-    $(`*[data-availability-record='true'] span`).text("Undetermined");
-    $(`*[data-availability-record='true'] span`).attr("title", "Cannot determine real-time availability for item at this time.");
-    $(`*[data-availability-record='true'] span`).addClass("badge badge-secondary");
+  // Sets the availability badge to indicate that we are retrying to fetch the information
+  update_av_retrying() {
+    var avBadges = $(`*[data-availability-record='true'] span.availability-icon`);
+    $(avBadges).text("Retrying...");
+    $(avBadges).attr("title", "Fetching real-time availability");
```

Comment:
Do we want the user to know that the code is retrying: `Retrying...` or is seeing `Loading...` is enough?

Reply:
Good idea. Done.